Makefile

```diff
@@ -26,7 +26,7 @@ vendor:
 IMAGE ?= quay.io/3scale/apicast-operator
 SOURCE_VERSION ?= master
 VERSION ?= v0.0.1
-NAMESPACE ?= operator-test
+NAMESPACE ?= $(shell kubectl config view --minify -o jsonpath='{.contexts[0].context.namespace}' 2>/dev/null || echo operator-test)
```

Comment:
Parsed from `current-context` string? Why take first context?

Reply:
--minify by definition will only return one context

```
      --minify=false: Remove all information not used by current-context from the output
```

Reply:
kubectl config current-context shows the context name and not the namespace. The context name is not the same as the namespace

Reply:
Hence me comment about `parsing from current-context string`. It is obvious that current-context contains more infor besides namespace.

minify flag does the trick,  indeed.

Reply:
> Hence me comment about parsing from current-context string. It is obvious that current-context contains more infor besides namespace.

That's not necessarily correct. The current context name can contain the namespace as part of the context name string or not